example/src/App.tsx

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
 import { StyleSheet, View, Text } from 'react-native';
-import { Popable } from 'react-native-popable';
+import { Popable, usePopable } from 'react-native-popable';
 
 export default function App() {
+  const [ref, { hide }] = usePopable();
+
   return (
     <View style={styles.container}>
-      <Popable content="See profile">
-        <Text>@eveningkid</Text>
+      <Popable ref={ref} content="See profile">
+        <Text onPress={hide}>@eveningkid</Text>
       </Popable>
     </View>
   );
```

Comment:
`usePopable` is not the favored way to use the library and therefore shouldn't be set as the example.

I think it's better to keep this either minimal or extensive (but this means we'd add more examples than this one).

So at the moment let's just keep it minimal.

However, **this makes me think that `usePopable` should be added to the `README`**.
It could be two small examples, one with show and another with hide (just like yours)

Reply:
Removed from the example and added to the readme.

Reply:
In the docs, I used inline functions for `show` and `hide` when passed to the `Menu` component. I find that this is simpler to newcomers in React/JS who don't always understand how callback functions work.

Reply:
Yes, all good man. You could even have passed the functions directly:
```jsx
onPress={() => show()}

// to

onPress={show}
```
Or is that what you meant by "callback functions"?

Either way, it's fine like this

Reply:
Yup that's what I'm referring to. Sometimes it's unclear to non-JS devs how we just pass functions around like that. I remember that problem when I first started using it. So I like to be explicit when documenting function props. But it's your call if you'd like to change it! Maybe we could have one of each in the examples.

Reply:
Well, I think it's okay this way. Maybe if I make other changes on the readme at some point, for now it's not necessary.

You know in some way I also feel that if you don't "teach", they might also think it works this way (because they've seen it in the documentation).

Anyways that's way too many thoughts for a popover library readme, they'll be fine

Reply:
Haha yeah, it's not that big of a deal.

You're right actually. That's a good point. It's better to show the right way.